core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -1417,6 +1417,12 @@ public MetricKey build() {
           .setMetricType(MetricType.GAUGE)
           .setIsClusterAggregated(false)
           .build();
+  public static final MetricKey CLIENT_CACHE_PAGE_SIZE =
+      new Builder("Client.CachePageSize")
+          .setDescription("Amount of pages used by the client cache.")
```

Comment:
As the description you gave, do you think the metric name `CLIENT_CACHE_PAGE_COUNT` can make sense more?